pkg/datapath/iptables/iptables.go

```diff
@@ -1036,6 +1036,17 @@ func (m *IptablesManager) InstallRules(ifName string) error {
 		}
 	}
 
+	// EKS expects the aws daemonset to set up specific rules for marking
+	// packets It marks packets coming from another node and restores the
```

Comment:
Nit: missing a period?

```suggestion
	// EKS expects the aws daemonset to set up specific rules for marking
	// packets. It marks packets coming from another node and restores the
```